### DISCUSSION TITLE
Too much dependencies on mac brew graphviz?

### DISCUSSION THREAD

**Discussion by reneix** [Homebrew/discussions#4839] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.15
ORIGIN: https://github.com/Homebrew/brew
HEAD: e57a87cfe9c15f192cd4fc6695a2b3b7acf0ddf7
Last commit: 2 days ago
Core tap JSON: 11 Oct 00:10 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: quad-core 64-bit ivybridge
Clang: 12.0.0 build 1200
Git: 2.24.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.7-x86_64
CLT: 12.4.0.0.1.1610135815
Xcode: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You are using macOS 10.15.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.
```

### Description of issue

hi there, i’m installing graphviz on mac by
brew install graphviz
however , the dependencies are too much as below, the installation not finished in 3 hours.
Any idea?

checked in graphviz forum,  
https://forum.graphviz.org/t/too-much-dependencies-on-mac-brew/1821

==> Fetching dependencies for graphviz: libpng, pkg-config, freetype, fontconfig, cmake, nasm, jpeg-turbo, brotli, giflib, highway, imath, xz, lz4, zstd, libtiff, little-cms2, openexr, webp, docbook, mpdecimal, ca-certificates, openssl@3, readline, sqlite, python@3.11, icu4c, boost, source-highlight, asciidoc, docbook-xsl, bison, gettext, flex, doxygen, docutils, python@3.12, python-setuptools, gdbm, python@3.10, pygments, python-certifi, python-flit-core, python-packaging, sphinx-doc, jpeg-xl, ninja, meson, libsodium, lua, ncurses, berkeley-db, perl, libyaml, m4, autoconf, libssh2, libgit2@1.6, six, z3, pcre2, swig, llvm@16, rust, ruby, vim, libvmaf, yasm, aom, libavif, gd, glib, apr, apr-util, utf8proc, scons, subversion, jasper, netpbm, automake, libtool, gts, util-macros, xorgproto, libxau, libxdmcp, xcb-proto, libxcb, xtrans, libx11, libxext, libxrender, lzo, pixman, cairo, gobject-introspection, gdk-pixbuf, fribidi, graphite2, harfbuzz, pango and librsvg

**Comment by p-linnane** [comment, marked as answer]
This is happening because you are on an unsupported OS, so you have to build graphviz from source. You should upgrade your Mac to macOS Monterey or newer if possible to take advantage of our bottles.

**Reply by reneix** [reply]
thanks, I will try it
